ci-conda.Dockerfile

```diff
@@ -273,14 +273,8 @@ EOF
 # Install codecov from source distribution
 ARG CODECOV_VER=notset
 RUN <<EOF
-# rust is needed for source builds of codecov-cli -- binaries are not available for Python 3.13 yet.
-# We must also remove rust in this step because it ships 750MB of documentation files.
-rapids-mamba-retry install -y rust
-# temporary workaround for discovered codecov binary install issue. See rapidsai/ci-imgs/issues/142
```

Comment:
This is an unrelated cleanup. We no longer need this workaround and I didn't want to expend the CI resources for a separate PR. I will note this in the PR description.